frontend/index.html

```diff
@@ -135,6 +135,54 @@ <h1 class="ubear-color-100">UBEAR</h1>
               >No visualisation</label
             >
           </div>
+
+          <div class="hotspots control">
+
+            <div class="form-group form-check">
+              <input
+                type="checkbox"
+                class="form-check-input"
+                id="hotspotsCheckbox"
+              />
+              <label class="form-check-label" for="hotspotsCheckbox"
+                >Enable hotspots</label
+              >
+            </div>
+
+            <div class="form-group">
+              <label for="hotspotDestUpdateNumber">Update number for creating a hotspot destination</label>
+              <input
+                type="number"
+                class="form-control"
+                id="hotspotDestUpdateNumber"
+                value="10"
+              />
+            </div>
+
+            <div class="form-group">
+              <label for="hotspotLocUpdateNumber">Update number for the hotspot come back</label>
+              <input
+                type="number"
+                class="form-control"
+                id="hotspotLocUpdateNumber"
+                value="30"
+              />
+            </div>
+
+            <div class="form-group">
+              <label for="hotspotPassNumber">Number of passengers for the hotspot</label>
+              <input
+                type="number"
+                class="form-control"
+                id="hotspotPassNumber"
+                value="10"
+              />
+            </div>
+
+            <div id="mapHotspotPositionRadBtn"></div>
```

Comment:
I think it's a better idea to move the radio button where the rest of radio buttons are